hw1/02-palindrome.js

```diff
@@ -1,3 +1,58 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput(elem) {
+  ///check for negative input
+  if (elem.target.value < 0) {
+    document.getElementById("result").innerHTML =
+      "That's a negative number. Please enter a positive number.";
+  }
+  //check for single digits
+  else if (elem.target.value < 10) {
+    document.getElementById("result").innerHTML = "Yes, this is a palindrome!";
+    document.getElementById("result").style.color = "green";
+  }
+  //check all positive numbers 2 digits or greater
+  else {
+    let query = elem.target.value;
+    let text = query.toString();
+    let textLen = text.length;
+
+    //check if there are an even number of digits and divisible by 11
+    if (textLen % 2 === 0 && query % 11 === 0) {
```

Comment:
I'm not sure if this really is necessary.  I guess it's one of the base case to check numbers like 11, 22, 33 but it doesn't really hurt to remove it.  The code below this will catch this case as well.
I think it helps with readability and confusion if we remove it but it's also just preference.